modules/ROOT/pages/policy-mule4-tokenization.adoc

```diff
@@ -0,0 +1,147 @@
+= Tokenization Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy Name | Tokenization.
+>s| Summary      | Transforms sensitive data into non-sensitive equivalent, named token. This way, the content is seen identical in format but its actual value is not revealed at first sight.
+>s| Category | Security.
+>s| Since Mule Version | 4.1.1
+.4+>.^s| Returned Status Codes |
+
+[%autowidth.spread,cols="a,a"]
+!===
+.2+>.^! 400 ! A token is invalid, or their tokenization failed.
+! The provided expression is invalid or returns no result to tokenize. +
+*NOTE*: The status code to be returned can be configured.
+! 502 ! Any problem regarding the connection with the Tokenization Service. It may be unavailable, or it might not have been configured.
+!===
+
+|===
+
+[NOTE]
+--
+This policy is designed to run only in a Runtime Fabric environment.
+--
+
+== Tokenization
+
+Tokenization is the process of transforming a value or piece of information (presumably sensitive data) into another one (i.e., the token), that can be mapped back onto the original value. +
+This tokenized information, replaces the original value with the token, thus minimizing the risk of matching it to the raw value in case someone happened to read it. For instance, it may be used to tokenize credit card numbers and map them to completely different ones from the originals.
+
+== Prerequisites
+
+* To create and apply policies you need the *Manage Policies* permission.
+* You need to have tokenization service up and running. +
+See xref:anypoint-security::tokenization-example.adoc[Configure and Use Tokenization] for more information.
+
+
+== How Does This Policy Work?
+
+The policy replaces some pieces of the payload with the tokenized values. To define which pieces of payload will be tokenized, you need to provide Dataweave 2.0 expressions.
+
+Suppose that the payload that will receive the policy contains the following structure:
+
+[source,JSON,linenums]
+--
+{
+  "name":"Amun Ra",
+  "socialSecurityNumber":"365-93-2738"
+}
+--
+
+If you want to tokenize the social security number, you can use the following Dataweave expression to specify which piece of information the policy should tokenize:
+
+
+[source,dataweave,linenums]
+--
+#[payload.socialSecurityNumber]
+--
+
+Now, every time the policy receives a request containing a `socialSecurityNumber` key in its top level, it will extract all the tokens that match the expression you indicated, it will establish a connection with the Tokenization Service, and finally it will replace the tokenized values in your payload with the ones provided by the service. +
```

Comment:
"it will replace the tokenized values in your payload with the ones provided by the service."

It actually replaces the value to be tokenized in your payload with the ones provided by the service.